### ISSUE TITLE
When I download the package, it throws a `husky install` error

### ISSUE THREAD

**Issue by liuqh0609** [sanyuan0704/island.js#58]
### Describe the bug

```shell
➜ pnpm i 
 WARN  deprecated hast@1.0.0: Renamed to rehype
Packages: +383
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 424, reused 383, downloaded 0, added 0, done
node_modules/.pnpm/islandjs@0.2.6/node_modules/islandjs: Running postinstall script, failed in 506ms
.../islandjs@0.2.6/node_modules/islandjs postinstall$ husky install
│ husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
└─ Failed in 506ms
 ELIFECYCLE  Command failed with exit code 1.
```
Maybe the script should write `npx husky install` like this? And I don't understand why I need to execute this command when installing dependencies.

### Reproduction

1. `pnpm i`

### Expected behavior

Correct installation of dependencies

### System Info

```Shell
System:
    OS: macOS 12.6
    CPU: (8) arm64 Apple M1
    Memory: 145.73 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.2 - /usr/local/bin/node
    Yarn: 1.22.18 - /usr/local/bin/yarn
    npm: 8.19.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 106.0.5249.91
    Safari: 16.0
  npmPackages:
    islandjs: ^0.2.6 => 0.2.6
```

### Additional context

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/43341115/193514900-ecc50d19-47c4-4fbc-98da-f3c6fd952b6d.png">

### Validations

- [X] Read the [docs](https://island.sanyuan0704.top)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by sanyuan0704** [comment]
The script should not be executed in project, it is only used in development environment.And i will remove it in `postinstall` script :)

**Comment by sanyuan0704** [comment]
I have fixed it in v0.2.7
